### PR TITLE
Add 1.20 headline features

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

**Review by lzap — APPROVED** [review]
TY

**Review by ohadlevy — APPROVED** [review]

File: _includes/manuals/1.20/1.2_release_notes.md [hunk 1]
```diff
@@ -4,7 +4,45 @@ This section will be updated prior to the next release.
...
+The purpose of this role is to set up environment for others to use. It can
+create organizations/locations but does not have access to the resources inside
+them. System admin can create new users and assign them to locations or
+organizations and add roles to the users.  See
+[Roles and Permissions](manuals/{{page.version}}/index.html#4.1.2RolesandPermissions)
+for more information.
+
+#### FIPS compliance
+
+Foreman should now run properly in FIPS-enabled environments.
```
Comment (ekohl) [inline _includes/manuals/1.20/1.2_release_notes.md:43]: Should sounds a bit weak, like we're not confident about it. How about `can`?

Comment (tbrisker) [reply _includes/manuals/1.20/1.2_release_notes.md:43]: I'll change that in the other PR, thanks.

Comment (lzap) [reply _includes/manuals/1.20/1.2_release_notes.md:43]: And you are confident? ;-)

Comment (ekohl) [reply _includes/manuals/1.20/1.2_release_notes.md:43]: I didn't test it but I'm confident in the developers who did :)

**Pull request merged**